mbed_app.json

```diff
@@ -1,109 +1,85 @@
 {
     "macros": [
         "MBED_TRACE_MAX_LEVEL=TRACE_LEVEL_INFO",
```

Comment:
```suggestion
```
This line can be removed now. Mbed OS6 has this enabled by default.

Reply:
Done